.azure-pipelines/nightly-CI.yml

```diff
@@ -70,15 +70,11 @@ jobs:
       testRunTitle: 'Python $(python.version)'
     condition: succeededOrFailed()
 
-- job: 'win32_conda_CI_nightly'
+- job: 'Win32'
```

Comment:
why does it have to be short?

Reply:
Makes it easier to view the whole name in the narrow Azure Pipelines side panel and see the Python version